packages/assets-controllers/src/NftController.ts

```diff
@@ -390,6 +403,17 @@ export class NftController extends BaseController<NftConfig, NftState> {
       tokenURI = getFormattedIpfsUrl(ipfsGateway, tokenURI, useIPFSSubdomains);
     }
 
+    if (!hasIpfsTokenURI && !displayNftMedia) {
```

Comment:
Nit: seems to be a lot of repetition here. Could we consolidate all of these cases where we want to return this null object?

Seems like it would be something like `!displayNftMedia && !(isIpfsGatewayEnabled && hasIpfsTokenURI)`

Reply:
Tested and will be pushed soon!

Thanks, it's really cleaner now